.travis/publish.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+echo "executing deploy.sh"
+echo "decrypting keyrings"
+openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/pubring.gpg.enc -out $GPG_DIR/pubring.gpg -d
+openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/secring.gpg.enc -out $GPG_DIR/secring.gpg -d
+
+if [ ! -z "$TRAVIS_TAG" ]
+then
+    echo "Checking out a tag $TRAVIS_TAG"
+    git checkout tags/$TRAVIS_TAG -b tag_$TRAVIS_TAG
+    mvn deploy --settings $GPG_DIR/settings.xml -DskipTests=true -B -U
```

Comment:
If this build fails for an external reason, like a network issue, can we re-run the build without pushing another tag?